.github/workflows/build.yml

```
@@ -30,7 +30,33 @@ jobs:
     - name: Build
       run: make
 
+  macos:
+    strategy:
+      matrix:
+        os: [ macos-latest, macos-arm64-latest ]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+    - uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)